suites/squid/nvmeof/tier-2_8-1_auto-load-balancing_tests.yaml

```diff
@@ -280,3 +280,84 @@ tests:
       module: test_ceph_nvmeof_loadbalancing.py
       name: One time GW scale up and scale down
       polarion-id: CEPH-83609608
+
+# 1 GWgroup 4GW 4-subsystems add and del NS
+  - test:
+      abort-on-fail: false
+      config:
+        rbd_pool: rbd2
+        gw_group: gw_group1
+        do_not_create_image: true
+        rep-pool-only: true
+        rep_pool_config:
+          pool: rbd2
+        install: true
+        cleanup:
+          - pool
+          - gateway
+          - initiators
+        gw_nodes:
+          - node4
+          - node5
+          - node6
+          - node7
+        subsystems:
+          - nqn: nqn.2016-06.io.spdk:cnode1
+            serial: 1
+            max_ns: 2048
+            bdevs:
+            - count: 20
+              size: 8G                              # auto NS load balancing for namespace addition would be verified here
+            listener_port: 4420
+            listeners: [node4, node5, node6, node7]
+            allow_host: "*"
+          - nqn: nqn.2016-06.io.spdk:cnode2
+            serial: 1
+            max_ns: 2048
+            bdevs:
+            - count: 20
+              size: 8G
+            listener_port: 4420
+            listeners: [node4, node5, node6, node7]
+            allow_host: "*"
+          - nqn: nqn.2016-06.io.spdk:cnode3
+            serial: 1
+            max_ns: 2048
+            bdevs:
+            - count: 20
+              size: 8G
+            listener_port: 4420
+            listeners: [node4, node5, node6, node7]
+            allow_host: "*"
+          - nqn: nqn.2016-06.io.spdk:cnode4
+            serial: 1
+            max_ns: 2048
+            bdevs:
+            - count: 20
+              size: 8G
+            listener_port: 4420
+            listeners: [node4, node5, node6, node7]
+            allow_host: "*"
+        initiators:
+          - nqn: connect-all
+            listener_port: 4420
+            node: node8
+        load_balancing:
+          - ns_del:
+              count: 10
+              subsystems:
+                - nqn.2016-06.io.spdk:cnode1
+                - nqn.2016-06.io.spdk:cnode2
+                - nqn.2016-06.io.spdk:cnode3
+                - nqn.2016-06.io.spdk:cnode4
+          - ns_del:
+              count: 5
```

Comment:
I think, you should test with deleting 3 namespaces to have uneven number.

Reply:
ok doing it with ns_add and ns_del with 3 namespaces